notifications_utils/recipients.py

```diff
@@ -289,15 +289,20 @@ def is_optional_address_column(self, key):
         )
 
     @property
-    def has_recipient_columns(self):
+    def required_recipient_column_headers(self):
         return set(
             Columns.make_key(recipient_column)
             for recipient_column in self.recipient_column_headers
             if not self.is_optional_address_column(recipient_column)
-        ) <= set(
-            self.column_headers_as_column_keys
         )
 
+    @property
+    def has_recipient_columns(self):
```

Comment:
Optional: Maybe this method name could be better, like `contains_required_recipient_column_headers`?

Reply:
I’d prefer to keep this as-is, because it would be a breaking change to rename it, and the current name matches other properties like `has_errors`.